AUTHORS

```diff
@@ -868,5 +868,125 @@ Super Zheng <superzheng@tencent.com>
 Rafał Pocztarski <r.pocztarski@gmail.com>
 Michael Ruddy <mruddybtc@gmail.com>
 Andy Bettisworth <andy.bettisworth@accreu.com>
+Jörg Krause <joerg.krause@embedded.rocks>
+Alexander Martin <alex@suitupalex.com>
+Prince J Wesley <princejohnwesley@gmail.com>
+janriemer <janriemer@users.noreply.github.com>
+Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
+Martin von Gagern <Martin.vGagern@gmx.net>
+Hideki Yamamura <hideki.yamamura@gmail.com>
+Lenny Markus <lmarkus@paypal.com>
+Nelson Pecora <nelson@yoshokatana.com>
+Xotic750 <xotic750@gmail>
```

Comment:
GitHub profile says _Graham Fairweather_. Also, the email is incorrect (misses the `.com`).
/cc @Xotic750